config/source.go

```diff
@@ -143,3 +146,30 @@ func ParseSource(cfgMap map[string]string) (Source, error) {
 
 	return cfg, nil
 }
+
+func (s *Source) setHelperObjects(cfgMap map[string]string) {
+	s.setDefaultHelperObjects()
```

Comment:
Sidenote: If we change the behavior to take these names from the position _only if_ the configuration doesn't specify them, then we'll have to know if the config was actually empty. No issue now, but we might have to change this in the future.

Reply:
Yes, you're correct about that. The reason why I put this here is that I wanted the result of `Parse` to be a complete configuration.